db/blip_sync_context.go

```diff
@@ -707,3 +712,13 @@ func (bsc *BlipSyncContext) reportStats(updateImmediately bool) {
 	bsc.stats.lastReportTime.Store(currentTime)
 
 }
+
+func (bsc *BlipSyncContext) reportComputeStat(rq *blip.Message, startTime time.Time) {
+	messageCPUtime := time.Since(startTime).Milliseconds()
+	messBody, err := rq.Body()
+	if err == nil && bsc.blipContextDb != nil {
+		bytes := len(messBody)
```

Comment:
if bytes is zero, let's early return here. I think you _might_ be able to have message with empty Body() and only Properties, but you can check that by constructing a message.